README.md

```diff
@@ -2,7 +2,7 @@
 
 # Portfolio: mobile version
 
-> This is the Third Project and i have added a portfolio section showing the projects worked on. 
+> This was one of my code project i worked on and it is going to be reviewed by new. 
 
 
 This is a Portfolio that works as a presentation card for a Software developer, showcasing some important projects, personal and contact info, while keeping a fresh and elegant web design. 
```

Comment:
Since the project does not have project work on yet and the contact form: It is better not to add them in the readme file description. This will be more clear and easier to understand even in the future.

Reply:
Hi, @ssenkooto1,   I am **Amine** your reviewer for this PR 😸 

Good job so far 👍. 
You did a great job detecting all bugs in the project

## STATUS: CHANGES REQUESTED ♻️

- [ ] Please ensure that you phrase your comments politely

_Please check the comments I left under your comments._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Reply:
Good comment👍.